kotlin/codegen/src/test/java/com/squareup/moshi/kotlin/codegen/KotlinCompilerCall.kt

```diff
@@ -140,7 +140,7 @@ class KotlinCompilerCall(var scratchDir: File) {
             ZipEntry("META-INF/services/${entry.key.qualifiedName}"))
         val serviceFile = Okio.buffer(Okio.sink(zipOutputStream))
         for (implementation in entry.value) {
-          serviceFile.writeUtf8(implementation.qualifiedName)
+          serviceFile.writeUtf8(implementation.qualifiedName!!)
```

Comment:
Why was it compiling before? You sure it's needed?

Reply:
was a warning before (because of ParametersAreNonNullByDefault in Okio 1.16.0)